packages/common/package.json

```
@@ -27,6 +27,10 @@
     "url": "https://github.com/AudiusProject/audius-client/issues"
   },
   "dependencies": {
+    "@tanstack/react-query": "^4.18.0",
+    "@trpc/client": "^10.38.4",
+    "@trpc/react-query": "^10.38.4",
```

Let's pin these bad boys:
```suggestion
    "@tanstack/react-query": "4.18.0",
    "@trpc/client": "10.38.4",
    "@trpc/react-query": "10.38.4",
```